jdaviz/configs/default/plugins/line_lists/line_lists.vue

```diff
@@ -248,6 +248,14 @@
                 ></v-text-field>
               </v-row>
 
+              <v-row>
+                <span class='text--primary' style="overflow-wrap: anywhere; font-size: 10pt;">
+                  <b>Medium: {{list_contents[item].medium}}</b>
+                </span>                    
+              </v-row>
```

Comment:
Custom lines end up with "Medium: "... should that either show "Medium: Unknown" or perhaps skip this block entirely?  (I think its overkill to have an input from the user, at least until we implement a switch for toggling).

Reply:
I added a custom medium message for the Custom list, specifically "Medium: Unknown (Custom)". That way it's clear to the user why

Reply:
ok, I think that's a good solution for now.  Definitely something we'll need to consider if/when we ever implement the toggle though 🤔